openapi/server.yml

```
@@ -0,0 +1,281 @@
+        authorization:
+          type: string
+          enum:
+            - none
+            - readonly
+            - full
+        userId:
+          type: string
+          nullable: true
+        metadata:
+          type: object
+              
```

yamllint

[error] trailing spaces

(trailing-spaces)